static/src/stylesheets/amp/_d2-comments.scss

```diff
@@ -0,0 +1,50 @@
+/* Comments
+   ========================================================================== */
+
+.d2-comment-embedded {
+    background-color: #ffffff;
+    font-family: sans-serif;
+    overflow: hidden;
+    position: relative;
+    margin-bottom: 1.5em;
+    padding-top: 15px;
```

Comment:
Thanks, thats good to know. I've had a crack at applying the grid values. It seems to look ok, but I'll run it by @zeftilldeath to be sure.